package.json

```diff
@@ -1,40 +1,32 @@
 {
   "name": "wagtail-readinglevel",
-  "version": "3.5.0",
+  "version": "3.6.0",
   "description": "Wagtail plugin that adds reading level calculations to Draftail.",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build-automated" : "webpack",
-    "build-smog" : "webpack --config webpack.smog.config.js",
-    "build-coleman" : "webpack --config webpack.coleman.config.js",
-    "build-flesch" : "webpack --config webpack.flesch.config.js",
-    "build-fog" : "webpack --config webpack.fog.config.js",
-    "build": "npm run build-automated && npm run build-smog && npm run build-coleman && npm run build-flesch && npm run build-fog"
+    "reading": "webpack --config webpack.readinglevel.config.js",
+    "smog": "webpack --config webpack.smog.config.js",
+    "coleman": "webpack --config webpack.coleman.config.js",
+    "flesch": "webpack --config webpack.flesch.config.js",
+    "fog": "webpack --config webpack.fog.config.js",
+    "build:prod": "npm run reading && npm run smog && npm run coleman && npm run flesch && npm run fog"
```

Comment:
The build:prod line gets quite long with the original names and I thought shorter is better. I'm happy to swap it back though as each of those stages does a build?

Reply:
Yep, I think the fuller name would be more descriptive here: I see what you mean about the build line getting quite long though - maybe it would help readability if that can be broken across lines.

Reply:
Thanks I have changed them here: https://github.com/torchbox-forks/wagtail-readinglevel/pull/6/commits/7b0c9a59cf85cff35ecf79213d958e7a4c16ac3b

Reply:
@alxbridge I have opened and issue to pick up adjusting the Readability Score later on.

Reply:
@alxbridge I think this PR should be good to go now. Can you give it your approval please, our new workflow requires an approval 😄 Thanks.